.vib/solr/cypress/cypress/integration/solr_test_suite.js

```diff
@@ -0,0 +1,96 @@
+/// <reference types="cypress" />
```

Comment:
[The name of the test should follow the following format (application_spec.js)](https://github.com/bitnami/charts/blob/master/TESTING.md#cypress)
P.S. I realised I made the same mistake with MiniO, which might have brought the confusion. Sorry about that!

Reply:
No problem, just renamed the file!